nixos/modules/programs/bash/bash.nix

```diff
@@ -33,7 +33,7 @@ let
   '';
 
   bashAliases = concatStringsSep "\n" (
-    mapAttrsFlatten (k: v: "alias ${k}=${escapeShellArg v}") cfg.shellAliases
+    mapAttrsFlatten (k: v: "alias ${k}=${escapeShellArg v}") (cfge.shellAliases // cfg.shellAliases)
```

Comment:
I think a better way to implement this is to have
```nix
{
  config.programs.bash.shellAliases = mapAttrs (name: mkDefault) cfge.shellAliases;
}
```

This should then have the `//` override behavior automatically through the module system. Also this allows the user to query all bash aliases by evaluating the option.

Reply:
Hmm... By your change, although defined in `/etc/nixos/configuration.nix`, bash aliases that have the same names as defined in `environment.shellAliases` seem not appearing in `programs.bash.shellAliases`.